helpers/image_helper.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'open-uri'
+require_relative '../config/StaticData'
+
+# Class for working with images after convert service
+class ImageHelper
+  def self.get_image_size(url)
+    temp_file = Tempfile.new('images-', StaticData::TMP_FOLDER.to_s, encoding: 'ascii-8bit')
+    temp_file.write URI.parse(url).open.read
```

Comment:
`URI.parse(url).open.read` seems that line will not close file after read
See:
https://stackoverflow.com/questions/8704751/does-rubys-open-uri-reliably-close-sockets-after-read-or-on-fail

`URI.parse(url).open {|f| f.read}` is safer